tests/render/renderer/v3/test_suite_profile_notebook_renderer.py

```diff
@@ -90,7 +90,11 @@ def test_suite_notebook_renderer_render_onboarding_data_assistant_configuration(
     # },
     # column_value_nonnullity_rule={
     # },
-    # numeric_columns_rule={
+    # numeric_columns_low_precision_rule={
```

Comment:
@NathanFarmer Please see the comment above regarding adding the rounding examples -- thank you!